### ISSUE TITLE
Axis labels are not rounded when using log-scale

### ISSUE THREAD

**Issue by Ezibenroc** [has2k1/plotnine#266]
The following code gives this result:
```python
import pandas
from plotnine import *
df = pandas.DataFrame({'x': [1.1**i for i in range(150)]})
df['y'] = 1.234e-10*df['x'] + 5.678e-6
ggplot(df, aes(x='x', y='y')) + geom_point() + scale_x_log10() + scale_y_log10()
```

![mwe](https://user-images.githubusercontent.com/8900834/54209709-88334280-44de-11e9-940f-f0bf89448722.png)

One of the values on the y axis is not rounded correctly.

I noticed that such problems happen frequently when using log scales. I have seen this on both the x and y axis. This does not happen when using a linear scale.

**Comment by has2k1** [comment]
This is a bug in the [mizani](https://github.com/has2k1/mizani).

**Comment by has2k1** [comment]
Fixed in mizani v0.5.4.
